website/public/js/controllers/rootCtrl.js

```diff
@@ -36,6 +36,8 @@ habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$
     $rootScope.toJson = angular.toJson;
     $rootScope.Payments = Payments;
 
+    $http.defaults.headers.common['x-client'] = "habitica-web";
```

Comment:
you're right, if it's not a big problem would you mind putting it in public/js/app.js like the other headers?